### DISCUSSION TITLE
Certificate verification issue while enabling metrics and trace type of log data for AKS cluster

### DISCUSSION THREAD

**Discussion by ajaykumarrc** [signalfx/splunk-otel-collector-chart#1087] [Q&A] [answered]
i have installed splunk otel connector on AKS cluster version **1.25.15**  after enabling metric type logs am getting the below error any workaround for the below error.

I can see logs coming for default logs but not for metrics or traces

`error   kubeletstatsreceiver@v0.91.0/scraper.go:77      call to /stats/summary endpoint failed  {"kind": "receiver", "name": "kubeletstats", "data_type": "metrics", "error": "Get \"https://1xx.2xx.xx.xxx:10250/stats/summary\": tls: failed to verify certificate: x509: cannot validate certificate for 1**.2***.**.**03 because it doesn't contain any IP SANs"}`

**Comment by dsentr** [comment, marked as answer]
In the CONTRIBUTING.md file at the bottom it gives you the overrides needed to fix this:

agent.config.receivers.kubeletstats.insecure_skip_verify = true
